src/dialogs/quick-bar/ha-quick-bar.ts

```diff
@@ -363,6 +362,12 @@ export class QuickBar extends LitElement {
 
         mwc-list-item {
           width: 100%;
+          --mdc-ripple-color: transparent;
+        }
+
+        [mwc-list-item]:not([selected]):hover,
+        [mwc-list-item]:not([selected]):focus {
+          background-color: var(--secondary-background-color);
         }
```

Comment:
```suggestion
          --mdc-ripple-color: var(--primary-color);
        }
```

Reply:
Not `--mdc-ripple-color: var(--primary-background-color);`?

Reply:
You wanted blue right? 😛

Reply:
Fixed globally to be a light gray #AAAAAA